WooCommerce/WooCommerceTests/POS/ViewModels/TotalsViewModelTests.swift

```diff
@@ -10,6 +25,25 @@ final class TotalsViewModelTests: XCTestCase {
     func test_formattedOrderTotalPrice() {}
     func test_formattedOrderTotalTaxPrice() {}
     func test_areAmountsFullyCalculated() {}
-    func test_clearOrder() {}
+    func test_clearOrder() {
+        // When
+        sut.clearOrder()
+
+        // Then
+        XCTAssertNil(sut.order)
+    }
     func test_setOrder() {}
+    func test_startNewTransaction() {
+        // Given
+
+        let paymentState: TotalsViewModel.PaymentState = .acceptingCard
+        // When
+
+        sut.startNewTransaction()
+
+        // Then
+        XCTAssertEqual(sut.paymentState, paymentState)
+        XCTAssertNil(sut.order)
+        XCTAssertNil(sut.cardPresentPaymentInlineMessage)
```

Comment:
To test if a new transaction sets this `nil`, we could have a test where  `cardPresentPaymentInlineMessage` gets set a different value and `startNewTransction` resets everything. For example,``test_startNewTransaction_when_payment_made_then_order_state_reset`